src/median.js

```diff
@@ -1,5 +1,31 @@
-export function median(data) {
+/* 
+ * Returns the meadian x value from a histogram of sorted abscissa
```

Comment:
There is a spelling mistake in 'meadian'. Please install the following vscode plugin to help you to avoid mistakes

![image](https://user-images.githubusercontent.com/1484241/117402495-cb46a000-af06-11eb-8360-43ad43359e39.png)